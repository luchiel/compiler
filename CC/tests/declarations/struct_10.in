struct a { struct a *h; } some_var;
struct b { struct b h; } some_var2;
