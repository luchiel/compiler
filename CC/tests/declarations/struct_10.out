type <int>
type <float>
type <void>
type <int*>
	pointer to <int>
type <void*>
	pointer to <void>
variable NULL of type <void*> initialized by
	{int 0}
		type <int>
type <0>
	struct 0
		tag a
		type <1>
			pointer to <0>
		variable h of type <1>
variable some_var of type <0>
type <2>
	struct 2
		tag b
		variable h of type <2>
variable some_var2 of type <2>
